Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -177,6 +191,16 @@ private static string CreateSerializedHeader()
             return NewtonsoftJsonSerializer.Instance.Serialize(header);
         }
 
+        /// <summary>Converts the PKCS8 PrivateKey to RSA Parameters. Uses the Bouncy Castle library.</summary>
+        private static RSAParameters ConvertPKCS8ToRSAParameters(string pkcs8PrivateKey)
+        {
+            Utilities.ThrowIfNullOrEmpty(pkcs8PrivateKey, "Pkcs8PrivateKey");
+            var base64PrivateKey = pkcs8PrivateKey.Replace("-----BEGIN PRIVATE KEY-----", "").Replace("\n", "").Replace("-----END PRIVATE KEY-----", "");

```

Comment:
I don't anticipate using this anywhere else. Do you want me to convert these to constants, even if these are used only here? I see that there are other string constants that are used inline in other places without declaring them as consts.